### DISCUSSION TITLE
problema con scaricamento video da MaxStream

### DISCUSSION THREAD

**Discussion by miottist68** [aclap-dev/video-downloadhelper#1547] [Help and Q&A]
### Your question

Buongiorno,
quando cerco di scaricare le puntate di una serie tv, il programma Video DownloadHelper scarica, uno dietro l'altro, frame di 20" e devo chiudere e riaprire sia il video che il programma svariate volte perchè intercetti il video e riesca a scaricarlo.
Fino a poco tempo fa, però, questo non succedeva.

Grazie

### Addon & CoApp version and details

version: 9.1.1.0
target: google
channel: stable
lang: it
coapp: {"found":true,"new_version":false,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19"}
license: {"accepted":true,"email":"<removed>","key":"33a4dbb44819403e"}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Safari/537.36
{
    "valid_license_message_has_been_displayed": true,
    "used_history_button": true,
    "download_directory": "C:\\Users\\miott\\Downloads",
    "never_show_no_incognito_msg_again": true,
    "view_options": {
        "all_tabs": false,
        "hide_downloaded": false,
        "low_quality": false,
        "show_button_clean": true,
        "show_button_clean_all": true,
        "show_button_convert_local": false,
        "sort_by_status": true,
        "sort_reverse": false
    },
    "use_sidebar": true,
    "open_count_store": 17,
    "successfull_dl": 99
}

**Comment by Mattmjs** [comment]
Buongiorno, where is the issue happening ? Do you have an example to share here.

**Reply by miottist68** [reply]
[errore Video Downloader.docx](https://github.com/user-attachments/files/18549573/errore.Video.Downloader.docx)

Buongiorno, allego quanto appare quando cerco di scaricare qualcosa .... i video di 20" sono in sequenza uno dietro l'altro e quello che sto scaricando lo devo cercare in mezzo a decine di video da 20" "inutili" .

Grazie

**Reply by TheHacker66** [reply]
@miottist68 Purtroppo i gestori di maxstream da anni le tentano tutte per cercare di evitare che la gente veda o scarichi video dalla loro piattaforma perchè sennò (dicono) non avrebbero più entrate dalle pubblicità.
Non è facile stargli dietro, e ovviamente chi gestisce la piattaforma può facilmente aggirare nuovi metodi di estrazione dei link diretti ai video.

**Reply by Mattmjs** [reply]
@miottist68 , I see from the attached file but it can't be investigated unless an example from maxstream is provided.

**Comment by Mattmjs** [comment]
@miottist68 , @TheHacker66 
I found an example, it detects the video without issue. The problem seems to occur if you don't have an adblocker installed.
Without the adblocker the menu starts populating with those small pieces of the file before even clicking play.
But with the adblocker installed, the menu doesn't fill with those small files. Once you click play the video is detected and you should
only see one entry in the menu.

**Reply by miottist68** [reply]
Buongiorno, ho attivato l'AdBlocker, ma il problema permane ... anzi, con l'AdBlocker attivato, il download è estremamente lento.

Grazie

**Comment by TheHacker66** [comment]
An adblock is mandatory nowadays

Il mar 28 gen 2025, 08:37 Mattmjs ***@***.***> ha scritto:
